components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/utils/RegistrySearchUtil.java

```diff
@@ -65,6 +65,7 @@
     private static final String PROVIDER_SEARCH_TYPE_PREFIX = "provider";
     private static final String VERSION_SEARCH_TYPE_PREFIX = "version";
     private static final String CONTEXT_SEARCH_TYPE_PREFIX = "context";
+    public static final String CONTEXT_TEMPLATE_SEARCH_TYPE_PREFIX = "contextTemplate";
```

Comment:
shall we make this private

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10929